src/templates/app/_layout/footer.jsx

```diff
@@ -18,6 +18,18 @@ const SocialIcons = ({ networks, is_centered }) => (
     </div>
 );
 
+const StatusNotification = () => (
+    <div id='status-notification'>
+        <div id='status-notification-type'>
+            <img src={it.url_for('images/server_status/ic-warning.svg')}  alt='information icon'/>
+        </div>
+        <p id='status-notification-text'/>
+        <div id='status-notification-close'>
+            <img src={it.url_for('images/server_status/ic-close.svg')}  alt='Warning icon'/>
```

Comment:
the alt here starts with an uppercase letter while on line 24 it's lowercase, better to keep it consistent